src/esp/metadata/attributes/ObjectAttributes.h

```diff
@@ -13,6 +13,31 @@ namespace esp {
 namespace metadata {
 namespace attributes {
 
+/**
+ * @brief This enum class defines the possible shader options for rendering
+ * instances of objects or stages in Habitat-sim.
+ */
+enum class ObjectInstanceShaderType {
+  /**
+   * Represents an unknown/unspecified value for the shader type to use. Resort
+   * to defaults for object type.
+   */
+  Unknown = -1,
+  /**
+   * Refers to flat shading, pure color and no lighting.  This is often used for
+   * textured objects
+   */
+  Flat = 0,
```

Comment:
Beware, our "flat-shaded" objects actually use a special material along with the Phong shader, if I'm not mistaken. ResourceManager::buildFlatShadedMaterialData .

Reply:
I think the purpose of calling these "flat" is to get away from specifying "no-lights" even though that is closer to the actual implementation.